apps/create-fuels-counter-guide/playwright.config.ts

```diff
@@ -29,14 +29,14 @@ export default defineConfig({
   ],
   webServer: [
     {
-      command: 'pnpm run original:dev',
+      command: 'pnpm run dev',
```

Comment:
Why was this command changed?

Reply:
This is indeed a mistake, thanks :) 

https://github.com/FuelLabs/fuels-ts/pull/3125/commits/40fdbeab709bc9b76aafb75a90adc974152c38e2